src/main/webapp/availabilityTable.jsp

```diff
@@ -26,12 +26,12 @@
     </tr>
   </thead>
   <tbody>
-    <!-- TODO: Allow clicking and scrolling over multiple slots to select them.-->
     <!-- TODO: Change page format so that it is vertically condensed.-->
     <c:forEach var = "i" begin = "0" end = "${pageScope.list.get(0).size() - 1}">
       <tr>
         <c:forEach items = "${pageScope.list}" var = "day">
-          <td onclick="toggleTile(this)" data-utc="${day.get(i).utcEncoding()}" 
+          <td onmouseenter="event.preventDefault(); toggleTile(this);" 
```

Comment:
I would need to test it, but it seems like this implementation would have the effect that if the user clicks on box A, then drags over B, C, and D, A will not be selected because the mouse was not down when A was entered.. Could you test that?

Reply:
Good catch, fixing now